ecs-agent/gmsacredclient/credentialsfetcher/credentialsfetcher.proto

```diff
@@ -15,6 +15,39 @@ service CredentialsFetcherService {
   rpc RenewNonDomainJoinedKerberosLease
       (RenewNonDomainJoinedKerberosLeaseRequest) returns (RenewNonDomainJoinedKerberosLeaseResponse);
   rpc DeleteKerberosLease (DeleteKerberosLeaseRequest) returns (DeleteKerberosLeaseResponse);
+  rpc HealthCheck(HealthCheckRequest) returns (HealthCheckResponse);
+  rpc AddKerberosArnLease (KerberosArnLeaseRequest) returns (CreateKerberosArnLeaseResponse);
+  rpc RenewKerberosArnLease (KerberosArnLeaseRequest) returns (RenewKerberosArnLeaseResponse);
+}
+
+message HealthCheckRequest {
+  string service = 1;
+}
+
+message HealthCheckResponse {
+  string status = 1;
+}
+
+message KerberosArnLeaseRequest {
+  repeated string credspec_arns = 1;
+  string access_key_id = 2;
+  string secret_access_key = 3;
+  string session_token = 4;
+  string region = 5;
+}
+
+message CreateKerberosArnLeaseResponse {
+  string lease_id = 1;
+  repeated KerberosTicketArnResponse krb_ticket_response_map = 2;
+}
+
+message RenewKerberosArnLeaseResponse {
+  string status = 1;
+}
+
+message KerberosTicketArnResponse {
+  string credspec_arns = 1;
```

Comment:
Are multiple ARNs stored in a single string?

Reply:
this is for future just in-case, we need to send out comma separated response for other distros